src/pip/_internal/commands/download.py

```diff
@@ -222,13 +232,39 @@ def run(self, options, args):
                 resolver.resolve(requirement_set)
 
                 downloaded = ' '.join([
-                    req.name for req in requirement_set.successfully_downloaded
+                    req.name
+                    for req in requirement_set.successfully_downloaded
```

Comment:
Unrelated change? I'm not too worried, but it's easier to review if you don't include unneeded cosmetic changes.